### PR TITLE
Wells API

### PR DESCRIPTION
**Pull request by stephenhillier**
Beginning of Wells API:
  * Boilerplate views/serializers for listing wells and list/create submissions
  * endpoints:  /api/v1/wells/ and /api/v1/submissions/
  * Postman API test collection for wells and submissions

### CODE REVIEW

File: app/wells/views.py [hunk 1]
```diff
@@ -15,16 +15,20 @@
...
 from django.views import generic
+from rest_framework.generics import ListAPIView, ListCreateAPIView
 
+from gwells import settings
 from gwells.models import Survey
 from gwells.roles import WELLS_ROLES
-from wells.models import Well
-from wells.documents import MinioClient
-from wells.permissions import WellsDocumentPermissions
+from registries.views import APILimitOffsetPagination
```
Comment (Sybrand) [inline app/wells/views.py]: As discussed, this should move to a common location - gwells.views

File: app/wells/views.py [hunk 2]
```diff
@@ -64,3 +68,70 @@ def get(self, request, tag):
...
+        qs = qs \
+            .select_related(
+                "bcgs_id",
+            ).prefetch_related(
+                Prefetch("water_quality_characteristics")
+            )
+        return qs
+
+    def list(self, request):
+        """ List wells with pagination """
```
Comment (Sybrand) [inline app/wells/views.py]: This should probably be ordered by something

File: app/wells/views.py [hunk 3]
```diff
@@ -64,3 +68,70 @@ def get(self, request, tag):
...
+    queryset = ActivitySubmission.objects.all()
+    pagination_class = APILimitOffsetPagination
+    serializer_class = WellSubmissionSerializer
+
+    def get_queryset(self):
+        qs = self.queryset
+        return qs
+
+    def list(self, request):
+        """ List wells with pagination """
```
Comment (Sybrand) [inline app/wells/views.py]: This should also probably be sored

**Comment by stephenhillier** [comment]
[EPIC#Edit | Implement API |5| !!!](https://trello.com/c/TpV24fX3/1138-epicedit-implement-api-5)

**Pull request merged**